### PR TITLE
Fix default themes

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/profile/ha-pick-theme-row.ts [hunk 1]
```diff
@@ -72,7 +67,8 @@ export class HaPickThemeRow extends LitElement {
         >
           <paper-listbox
             slot="dropdown-content"
-            .selected=${this._selectedThemeIndex}
+            .selected=${this.hass.selectedTheme?.theme || "Backend-selected"}
```
[the remark this answers is not in the document]
Comment (bramkragten) [reply src/panels/profile/ha-pick-theme-row.ts:70]: Not for this PR

File: src/state/themes-mixin.ts [hunk 2]
```diff
@@ -74,16 +74,15 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
...
           : this.hass.themes.default_theme);
 
       let darkMode =
         themeSettings?.dark === undefined ? darkPreferred : themeSettings?.dark;
 
-      const selectedTheme =
-        themeSettings?.theme !== undefined
-          ? this.hass.themes.themes[themeSettings.theme]
-          : undefined;
+      const selectedTheme = themeName
```
Comment (balloob) [inline src/state/themes-mixin.ts:83]: ==

File: src/panels/profile/ha-pick-theme-row.ts [hunk 3]
```diff
@@ -72,7 +67,8 @@ export class HaPickThemeRow extends LitElement {
...
+            attr-for-selected="theme"
```
Comment (balloob) [inline src/panels/profile/ha-pick-theme-row.ts:71]: Theme is a prop, not attr. Is that ok?

Comment (bramkragten) [reply src/panels/profile/ha-pick-theme-row.ts:71]: Yep

File: src/state/themes-mixin.ts (hunk 2 above)
Comment (bramkragten) [reply src/state/themes-mixin.ts:83]: ?

Comment (balloob) [reply src/state/themes-mixin.ts:83]: Nm, read the line wrong

**Review by balloob — APPROVED** [review]

**Pull request merged**